static/app/types/index.tsx

```diff
@@ -1978,11 +1978,22 @@ export type Frame = {
   symbolicatorStatus: SymbolicatorStatus;
   trust: any | null;
   vars: Record<string, any> | null;
+  addrMode?: string;
```

Comment:
I think this fits well with instructionAddr, it's topically related

Reply:
this prop was just moved to the bottom of the object since it's optional